src/db.rs

```diff
@@ -98,7 +92,7 @@ impl KbsDb {
         let mut counter = 0;
         let result = question_mark_re.replace_all(sql, |_: &Captures| {
             counter += 1;
-            format!("${}", counter)
+            format!("${counter}")
```

Comment:
is this `$` sign needed?

Reply:
Postgres variable bindings (simple `?` in mysql land) are `$1`, `$2`, `$3` , etc.

Reply:
ah, ok. thanks @dubek !